### PR TITLE
Including ops_partition("") call

### PR DESCRIPTION
**Pull request by maelso**
This commit is including the ops_partition("") call into code generated by Devito using OPS backend.
The class Literal was created in the codebase.

### CODE REVIEW

**Comment by vincepandolfo** [comment]
LGTM

File: devito/symbolics/extended_sympy.py [hunk 1]
```diff
@@ -357,6 +357,14 @@ class Macro(sympy.Symbol):
     pass
 
 
+class Literal(sympy.Symbol):
```
Comment (mloubout) [inline devito/symbolics/extended_sympy.py]: Why is this needed? Can't you use the `Symbol` class in devito.types?

Comment (vincepandolfo) [reply devito/symbolics/extended_sympy.py]: Because of the check in `utils.py` I believe.

Comment (mloubout) [reply devito/symbolics/extended_sympy.py]: I get that the check needs an extra, but I don't see why it needs to create a new (mostly copy) of an already existing class. just use `devito.symbol` and same in the check in utils.

Comment (vincepandolfo) [reply devito/symbolics/extended_sympy.py]: You can't use Symbol in that check though, it would break using symbols as arguments

Comment (maelso) [reply devito/symbolics/extended_sympy.py]: Symbol works, but I could break other locations. Here I am creating a type for this specific case.

**Comment by maelso** [comment]
I can't understand why the travis is breaking.

**Comment by FabioLuporini** [comment]
yeah, that's really odd... I'll restart it, no idea what's going on. pipelines seems fine at least

**Comment by FabioLuporini** [comment]
Merged.
